### DISCUSSION TITLE
2TB * 2 Total 4 TB is used as Parity ?

### DISCUSSION THREAD

**Discussion by pilgrimage80** [trapexit/mergerfs#1187] [General]
Hello, I have 4 4 TB disks. I have 2 2*2 TB disks.

Could there be such a scenario? 

4TB*1 Data Content
4TB *1 Data Content
4TB *1 Data Content
4 TB *1 Data Content
1 TB *1 Data Content
2TB*1 Parity
2TB*1 Parity 

What is the most appropriate and logical scenario?

**Comment by trapexit** [comment]
You are asking about snapraid? The best thing would be to talk to snapraid people.  But yes... your parity needs to be at least as large as the largest data drive.
